public/app/plugins/datasource/elasticsearch/tracking.ts

```diff
@@ -121,6 +121,7 @@ export function trackQuery(response: DataQueryResponse, queries: ElasticsearchQu
     }
     reportInteraction('grafana_elasticsearch_query_executed', {
       app,
+      grafana_version: config.buildInfo.version,
       with_lucene_query: query.query ? true : false,
       query_type: getQueryType(query),
       line_limit: getLineLimit(query),
```

Comment:
We could also add here the same props (besides bytes):
```
time_range_from: request?.range?.from?.toISOString(),
time_range_to: request?.range?.to?.toISOString(),
time_taken: Date.now() - startTime.getTime(),
```

Reply:
Good idea, added in a704feec6e04157a015d545141d5aef841ba443f